src/Symfony/Component/Uid/Ulid.php

```diff
@@ -79,7 +79,7 @@ public static function fromString(string $ulid): parent
             base_convert(substr($ulid, 27, 5), 16, 32)
         );
 
-        return new self(strtr($ulid, 'abcdefghijklmnopqrstuv', 'ABCDEFGHJKMNPQRSTVWXYZ'));
+        return new static(strtr($ulid, 'abcdefghijklmnopqrstuv', 'ABCDEFGHJKMNPQRSTVWXYZ'));
```

Comment:
This place is only reached if the input is binary, the test needs to reflect that.

Reply:
Done